inlong-tubemq/tubemq-manager/pom.xml

```diff
@@ -176,6 +176,10 @@
             <groupId>jakarta.validation</groupId>
             <artifactId>jakarta.validation-api</artifactId>
         </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-common</artifactId>
```

Comment:
Excuse me, where did you use the netty-common package?

Reply:
The DefaultThreadFactory in TopicBackendWorker.java